app/src/main/java/org/mozilla/fenix/settings/address/AddressEditorFragment.kt

```diff
@@ -26,6 +30,15 @@ class AddressEditorFragment : SecureFragment(R.layout.fragment_address_editor) {
 
     private lateinit var addressEditorView: AddressEditorView
     private lateinit var interactor: AddressEditorInteractor
+    private val args by navArgs<AddressEditorFragmentArgs>()
```

Comment:
Can you add a new line before? I prefer to just keep `lateinit var` and `val` spatially separated for easy parsing